### PR TITLE
Create create-account.md

### PR DESCRIPTION
**Pull request by kpslp**
create acc

docs to markdown w no edits yet

### CODE REVIEW

**Comment by kpslp** [comment]
@SenthilKumarSivakami updated docs. as the docs had many comments, it was hard for me to read the comments. this is first cut of staging draft. will edit more, ie to make it more coherent for the information between steps and table.

made approval to have screenshots and the create account not to have screenshots to see what would work better.

File: create-account.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    - **Agency department** name, **Cost centre** , and **Fund centre**. (For GovTech only)
+  - Subscription plans for the required SGTS product(s).
+  - Supporting documents. (Optional)
+
+**To create account**
+
+1. In **Overview**, click **Manage accounts**.
+
+2. Click **Create account**.
+
```
Comment (SenthilKumarSivakami) [inline create-account.md:32]: @kpslp missing a step

File: create-account.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+3. Agree to the terms and conditions of the **GovTech Universal Service Terms** and **SGTS Service Sheet**.
+
+4. To download the agreement, click **Download the SGTS agreement** link.
```
Comment (SenthilKumarSivakami) [inline create-account.md:35]: swap the steps.

File: create-account.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+| **Field Name** | **Description** |
+| --- | --- |
+| **Agency name** | Select your agency name from the drop-down list. |
+| **Account name** | Enter the account name. Account name can contain alphanumeric characters. |
+| **Account ID** | A unique **Account ID** is automatically generated and cannot be edited. Your **Account ID** is common across all SGTS products that your agency has subscribed to under this account.
+
+> **Notes:**
+> - All fields are mandatory, unless mentioned otherwise.
+> - Click **Save draft** at any point before submitting your request.
```
Comment (SenthilKumarSivakami) [inline create-account.md:51]: why should i  click save draft?

**Pull request closed without merging**